src/store/userModule.js

```diff
@@ -0,0 +1,21 @@
+import router from "../router";
+
+export const user = {
+    state: () => ({
+        isLogged: false,
+    }),
+    mutations: {
+        loginUser (state) {
+            state.isLogged = true
+            router.push("overview")
```

Comment:
another thing is, the mutation should not have additional side effects. If you would to push to some view, do it in action